src/PKSim.Assets/PKSimConstants.cs

```diff
@@ -1103,7 +1103,9 @@ public static class MenuNames
          public static readonly string NewFormulation = "Add &Formulation...";
          public static readonly string NewEvent = "Add &Event...";
          public static readonly string AddObservedData = "Add &Observed Data...";
-         public static readonly string AddObservedDataFor = "Add Observed Data for";
```

Comment:
Maybe we should rename all the observed to concentration?
Add Concentration Data and Add Amount Data as opposed to Observed ?

Reply:
I would vote for "Add concentration observed data". Let's open an issue for this and ask users.

Reply:
>"Add concentration observed data
That's a mouth full.
An alternative is to have  Add Observed Data and then a sub selection => Concentration, Amount., Fraction

But then the Add Observed Data For gets more comlicated?

Reply:
Let's open a separate issue on naming. Would like to hear some more opinions from the users.

Reply:
done 
https://github.com/Open-Systems-Pharmacology/Forum/issues/245